symengine/functions.h

```diff
@@ -1255,6 +1267,8 @@ RCP<const Basic> min(const vec_basic &arg);
 //! \return simplified form if possible
 RCP<const Basic> trig_to_sqrt(const RCP<const Basic> &arg);
 
+RCP<const Basic> expand_as_exp(const RCP<const Basic> &self);
```

Comment:
Similarly can it be added to `TrigFunction` / `HyperbolicFunction` as a virtual function?